src/test/scala/edu/ie3/simona/agent/grid/DBFSGridAgents.scala

```diff
@@ -0,0 +1,146 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.agent.grid
+
+import akka.actor.ActorRef
+import akka.testkit.TestProbe
+import edu.ie3.simona.ontology.messages.PowerMessage.ProvideGridPowerMessage.ExchangePower
+import edu.ie3.simona.ontology.messages.PowerMessage.{
+  ProvideGridPowerMessage,
+  RequestGridPowerMessage
+}
+import edu.ie3.simona.ontology.messages.VoltageMessage.ProvideSlackVoltageMessage.ExchangeVoltage
+import edu.ie3.simona.ontology.messages.VoltageMessage.{
+  ProvideSlackVoltageMessage,
+  RequestSlackVoltageMessage
+}
+import edu.ie3.simona.test.common.UnitSpec
+
+import java.util.UUID
+import scala.concurrent.duration.DurationInt
+
+trait DBFSGridAgents extends UnitSpec {
```

Comment:
`DBFSMockGridAgents` would be a more expressive name. Also, ScalaDoc for this trait (short description) would be nice